<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Искать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Товар</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Кол-во запросов</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Файлы для загрузки</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <widget class="QTableWidget" name="tableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Наименование поставщика</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Почта</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Адрес</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ИНН</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Печатная форма</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>номер</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Регион</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Товары </string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
